src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1783,6 +1783,23 @@ public List<Setting<?>> getSettings() {
                     Property.Filtered
                 )
             );
+            settings.add(
+                Setting.boolSetting(
+                    ConfigConstants.SECURITY_SYSTEM_INDICES_ADDITIONAL_CONTROL_ENABLED_KEY,
+                    ConfigConstants.SECURITY_SYSTEM_INDICES_ADDITIONAL_CONTROL_ENABLED_DEFAULT,
+                    Property.NodeScope,
+                    Property.Filtered
+                )
+            );
+            settings.add(
+                Setting.listSetting(
+                    ConfigConstants.SECURITY_INDICES_DENYLIST_KEY,
```

Comment:
This should not be a configurable setting, lets delete pulling this from configuration